timemachine/fe/interpolate.py

```diff
@@ -162,6 +162,7 @@ def log_linear_interpolation(src_params, dst_params, lamb, min_value):
     src_params = jnp.maximum(src_params, min_value)
     dst_params = jnp.maximum(dst_params, min_value)
 
+    # tbd: handle 0s better if lamb = 0 and src_params == 0
```

Comment:
if `src_params == 0 and lamb == 0`, then technically we have: `exp(log(0)) == 0`

Reply:
hmm, but isn't this handled by `src_params = jnp.maximum(src_params, min_value)` on L162?

Reply:
if `min_value==0` does it get handled?

Reply:
(sorry was playing with the setting of `min_value=0`, maybe that's not supported by this method).

Reply:
> if min_value==0 does it get handled?

No, but the original intent of `min_value` was as a positive threshold. Maybe we should add an assertion that `min_value > 0`?

Reply:
> (sorry was playing with the setting of `min_value=0`, maybe that's not supported by this method).

ah, I see - yeah, interpolation of functions with zero or negative values is not supported by this method. In principle it could be if we extended the functional form, e.g. to add an offset